component/src/main/java/io/siddhi/extension/store/elasticsearch/ElasticsearchConfigs.java

```diff
@@ -0,0 +1,532 @@
+                ioThreadCount = Integer.parseInt(
+                        configReader.readConfig(ANNOTATION_ELEMENT_CLIENT_IO_THREAD_COUNT,
+                                String.valueOf(ioThreadCount)));
+            }
+            if (!ElasticsearchTableUtils.isEmpty(storeAnnotation.getElement(ANNOTATION_ELEMENT_SSL_ENABLED))) {
+                sslEnabled = Boolean.parseBoolean(storeAnnotation.getElement(ANNOTATION_ELEMENT_SSL_ENABLED));
+            } else {
+                sslEnabled = Boolean.parseBoolean(configReader.readConfig(ANNOTATION_ELEMENT_SSL_ENABLED,
+                        String.valueOf(sslEnabled)));
+            }
+            if (!ElasticsearchTableUtils.isEmpty(storeAnnotation.getElement(ANNOTATION_ELEMENT_TRUSRTSTORE_PASS))) {
+                trustStorePass = storeAnnotation.getElement(ANNOTATION_ELEMENT_TRUSRTSTORE_PASS);
+            } else {
+                trustStorePass = configReader.readConfig(ANNOTATION_ELEMENT_TRUSRTSTORE_PASS, trustStorePass);
+            }
+            if (!ElasticsearchTableUtils.isEmpty(storeAnnotation.getElement(ANNOTATION_ELEMENT_TRUSRTSTORE_PATH))) {
+                trustStorePath = storeAnnotation.getElement(ANNOTATION_ELEMENT_TRUSRTSTORE_PATH);
+            }
+            if (!ElasticsearchTableUtils.isEmpty(storeAnnotation.getElement(ANNOTATION_ELEMENT_TRUSRTSTORE_TYPE))) {
+                trustStoreType = storeAnnotation.getElement(ANNOTATION_ELEMENT_TRUSRTSTORE_TYPE);
+            } else {
+                trustStoreType = configReader.readConfig(ANNOTATION_ELEMENT_TRUSRTSTORE_TYPE, trustStoreType);
+            }
+            if (!ElasticsearchTableUtils.isEmpty(storeAnnotation.getElement(ANNOTATION_ELEMENT_MEMBER_LIST))) {
+                listOfHostnames = storeAnnotation.getElement(ANNOTATION_ELEMENT_MEMBER_LIST);
+            }
+        } else {
+            throw new SiddhiAppCreationException("Elasticsearch Store annotation list null for " + type +
+                    " id : '" + definition.getId() + "', required properties cannot be resolved.");
+        }
+        CredentialsProvider credentialsProvider = new BasicCredentialsProvider();
+        credentialsProvider.setCredentials(AuthScope.ANY,
+                new UsernamePasswordCredentials(userName, password));
+        HttpHost[] httpHostList;
+        if (listOfHostnames != null) {
+            String[] hostNameList = listOfHostnames.split(",");
+            httpHostList = new HttpHost[hostNameList.length];
+            for (int i = 0; i < httpHostList.length; i++) {
+                try {
+                    URL domain = new URL(hostNameList[i]);
+                    httpHostList[i] = new HttpHost(domain.getHost(), domain.getPort(), domain.getProtocol());
+                } catch (MalformedURLException e) {
+                    throw new SiddhiAppCreationException("Provided elastic search hostname url list is " +
+                            "malformed of " + type + " id : '" + definition.getId() + ".", e);
```

Comment:
Add the url list to the error message.

Reply:
Fixed in https://github.com/siddhi-io/siddhi-store-elasticsearch/pull/41/commits/2c90b5856b203a9ad479c745f00b96442646fa04